src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -34,50 +37,147 @@ public Vision()
         RobotLog.getInstance().log("Created Vision Subsystem");
     } 
 
-    public boolean frontCamHasTargets()
+    public PhotonPipelineResult getFrontResult()
     {
-        return _frontLatestResult.hasTargets();
+        return _frontLatestResult;
     }
-    
-    public boolean rearCamHasTargets()
-    {   
-        return _rearLatestResult.hasTargets();
+
+    public PhotonPipelineResult getRearResult()
+    {
+        return _rearLatestResult;
+    }
+
+    private double getTargetHeight(int id)
```

Comment:
The WPILibrary has an `AprilTagFieldLayout` class that has all of these numbers already. Is there a reason we're not using that and instead doing this?

Reply:
I didn't want to deal with scaling and translate the meters measured from the corner to a format that's usable in inches. It was much easier to just define constants for it on my own.